index.html

```diff
@@ -156,6 +156,40 @@ <h2 class="titles-about-me">About me</h2>
       </ul>
     </article>
   </section>
-    
+  <footer class="footer">
+    <p class="contact-par">I'm always interested in hearing about new projects,
+      so if you'd like to chat please get  in touch.</p>
+    <form action="https://formspree.io/f/mzbwkgww" method="post"class="form">
+      <input type="text"
+      placeholder="Full name"
+      required
+      maxlength="30"
+      name="name"
+      class="personal-info">
+      <input type="email"
+      placeholder="Email address"
+      required
+      maxlength="30"
```

Comment:
- [ ] there's no need to add `maxlength="30"` to the email field since some emails can have more than 30 characters and since its not part of the requirements kindly remove that HTML validation for the email field 🙏